.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,51 +1,36 @@
-## Describe the PR
+## This PR …
 <!--
-A clear and concise description of the bug the PR fixes or the feature the PR introduces.
-Use this section for review hints and explanations for easier code understanding if necessary.
--->
-
+A clear and concise description of the PR introduces.
```

Comment:
Something is wrong with this sentence? Shouldn't this just read "A clear and concise description of the PR."?

Reply:
Good catch.